golem_core/golem_node.py

```diff
@@ -21,6 +21,10 @@
 from .low.payment_event_collector import DebitNoteEventCollector, InvoiceEventCollector
 
 
+DEFAULT_DRIVER: str = os.getenv("YAGNA_PAYMENT_DRIVER", "erc20").lower()
+DEFAULT_NETWORK: str = os.getenv("YAGNA_PAYMENT_NETWORK", "rinkeby").lower()
+DEFAULT_SUBNET: Optional[str] = os.getenv("YAGNA_SUBNET", "public")
```

Comment:
Looks like default values can be changed? Not good idea. Let's investigate, where what values are used, and rework that logic. Optionally introduce some function to resolve that values, using under the hood env lookup with default value fallback.

Reply:
Rename to 
```python
PAYMENT_DRIVER: str = os.getenv("YAGNA_PAYMENT_DRIVER", "erc20").lower()
PAYMENT_NETWORK: str = os.getenv("YAGNA_PAYMENT_NETWORK", "rinkeby").lower()
SUBNET: Optional[str] = os.getenv("YAGNA_SUBNET", "public")
```